src/pages/AboutPage.js

```diff
@@ -1,19 +1,23 @@
 import React from 'react';
 import { Button, Col, Container, Image, Row } from 'react-bootstrap';
 import { LinkContainer } from 'react-router-bootstrap';
-import logo from '../media/blm.png';
+import logo from '../assets/logo.png';
+import commPartnersIcon from '../assets/community-partners.png';
+import reqProjectIcon from '../assets/form-icon.png';
+import workProjectIcon from '../assets/project-selct-icon.png';
+import mailIcon from '../assets/email-icon.png';
 import fbLogo from '../assets/social_logos/Facebook_Logo.png';
 import igLogo from '../assets/social_logos/IG_Logo.png';
 import linkedinLogo from '../assets/social_logos/Linkedin_Logo.png';
 import twitterLogo from '../assets/social_logos/Twitter_Logo.png';
 
 const AboutPage = () => (
-  <Container className="container" fluid="md">
-    <Row>
+  <Container className="container" fluid>
+    <Row className="align-center">
       <Col>
```

Comment:
To make it better for mobile, you can try adding `xs={12} md={6}` to your Cols and media-query css to add a margin bottom to the image when it gets smaller. You can look at my latest homepage.css to see what i'm talking about!

Reply:
I would also change the size of the images at different breakpoints (using the media-queries) so it doesn't overwhelm the text and seem too big.

Reply:
Did a little research, seems like you can rearrange the order using CSS at different screen widths too!
<img width="120" alt="Screen Shot 2020-07-04 at 11 29 08 PM" src="https://user-images.githubusercontent.com/43453743/86524892-2d012e00-be4e-11ea-82b3-8ed993967b7e.png">
Would look something like this: `@media screen and (max-width: 768px) {
  .child1 {
    order: 2;
  }
  .child2 {
    order: 1;
  }
}`

Reply:
adjusted some things! maybe a tad too spaced out on smaller screens (against the hr, at least), but seems much better than before

Reply:
Looks great! Could you please apply this CSS: `@media screen and (max-width: 768px) { .child1 { order: 2; } .child2 { order: 1; } }` so that the image appears above the section for Community Partners and Work on a Project?

Reply:
didn't see this comment before - working on it rn!

Reply:
seemed to work with just modifying one child, so i did that